nmhieu/Capstone/Capstone/Resource/NSObject+Extension.swift

```diff
@@ -0,0 +1,14 @@
+//
+//  NSObject+Extension.swift
+//  Capstone
+//
+//  Created by Nguyễn Minh Hiếu on 02/07/2021.
+//
+
+import Foundation
+
+extension NSObject {
+    class var name: String {
```

Comment:
e không nên quá lạm dụng extension trong case này việc ép kiểu khá đơn giản và có thể xử lí trong 1 line code không cần thiết thêm extension. Sẽ làm mất thời gian cho những người đọc code về sau vì phải 'jump to definition' để xem.